api/controllers/pickController.js

```diff
@@ -8,6 +8,34 @@ const getPick = asyncErrorHandler(async (request, response) => {
   return response.status(200).json({ data: results });
 });
 
+const postPick = asyncErrorHandler(async (request, response) => {
+  const userId = request.userId;
+  const { id, counts, price } = request.body;
+  if (!userId || !id || !counts || !price)
+    throwCustomError("POSTING PICK ERROR", 400);
+  await pickService.postPick(userId, id, counts, price);
+  return response.status(200).json({ message: "PICK POSTED" });
```

Comment:
```suggestion
  const userId = request.userId;
  const { id, counts, price } = request.body;
  
  if (!userId || !id || !counts || !price)
    throwCustomError("POSTING PICK ERROR", 400);
    
  await pickService.postPick(userId, id, counts, price);
  
  return response.status(201).json({ message: "PICK POSTED" });
```
코드의 의미에 맞춰 줄바꿈을 해야 가독성이 좋을 것 같아요.

그리고 장바구니 담기 기능이면 데이터 생성에 적합한 status code를 작성하는 것이 좋겠습니다.